UnitsNet.NumberExtensions.CS14/UnitsNet.NumberExtensions.CS14.csproj

```diff
@@ -23,16 +23,14 @@
     <VersionNoSuffix>$([System.Text.RegularExpressions.Regex]::Match($(Version), '^[0-9.]+')).0</VersionNoSuffix>
     <AssemblyVersion>6.0.0.0</AssemblyVersion>    <!-- Fixed to major version, for strong naming -->
     <FileVersion>$(VersionNoSuffix)</FileVersion> <!-- Match the NuGet version number, except any suffix in the semantic version. -->
-    <LangVersion>preview</LangVersion>
+    <LangVersion>latest</LangVersion>
     <Nullable>enable</Nullable>
     <RootNamespace>UnitsNet</RootNamespace>
-    <TargetFrameworks>netstandard2.0;net8.0;net9.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net8.0;net9.0;net10.0</TargetFrameworks>
 
-    <!-- TEMPORARY WORKAROUND UNTIL NET10 SDK IS STABLE
-         Disable ILLink analyzers due to incompatibility with C# 14 extension members preview
+    <!-- Disable ILLink analyzers due to incompatibility with C# 14 extension members preview
          "CSC : error AD0001: Analyzer 'ILLink.RoslynAnalyzer.DynamicallyAccessedMembersAnalyzer' threw an exception of type 'System.InvalidCastException' with message 'Unable to cast object of type 'Microsoft.CodeAnalysis.CSharp.Symbols.PublicModel.NonErrorNamedTypeSymbol' to type 'Microsoft.CodeAnalysis.IMethodSymbol'.'." -->
-    <IsAotCompatible>false</IsAotCompatible>
-    <!-- <IsAotCompatible Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net7.0'))">true</IsAotCompatible> -->
+    <IsAotCompatible Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net10.0'))">true</IsAotCompatible>
```

Comment:
I just realized that the persistence of this `ILink.RoslyAnalyzer` exception is likely related to this issue: https://github.com/dotnet/runtime/issues/120728. That's a bit of a bummer, as if we wanted to use the extension features in any of the other projects (not just the extension properties) we'd be forced to push the AoT compatibility version to `net10.0`